server/routes/assignOnCallsTest.test.js

```diff
@@ -0,0 +1,25 @@
+const { JsonWebTokenError } = require('jsonwebtoken');
+
+const filterUsingPeriods = require('./assignOnCalls').filterUsingPeriods;
+const filterUsingTeachables = require('./assignOnCalls').filterUsingTeachables;
+const getAvailability = require('../routes/getAvailability').getAvailablePeriods;
+const getClassesToBeCovered = require('../routes/getClassesToBeCovered').getClassesToBeCovered;
+const getMonthlyOnCalls =  require('../routes/assignOnCalls').getMonthlyOnCalls;
+
+test ('filter using periods', async function () {
+    jest.setTimeout(10000);
+    let date = new Date ("2022-02-15");
+    const teachers = await getAvailability(date);
+    const classes = await getClassesToBeCovered(date);
+    console.log('I AM HERE!!!!!')
+    console.log(classes);
+    const filterUsingPeriod = filterUsingPeriods(classes[0], teachers);
```

Comment:
Looks like this test isnt passing since classes is just an empty array. Could be because theres no data in the database? 
Try adding this to the test file: 
![Screenshot 2022-12-01 124210](https://user-images.githubusercontent.com/97375107/205110122-1c626a83-40e0-424d-8fa3-f4d6c361f808.png)

Reply:
i just used this to test methods while I was working on the ticket, should I just delete this file?

Reply:
All the functions that takes in a date is suppose to get the current date (2022-12-01). I think its empty cause there are no classes that needs to be covered today.

Reply:
I think I should just delete this file cause we literally have like an hour left

Reply:
im scared i will break something last min ahhh

Reply:
Okay yeah just delete haha

Reply:
You might have to pull in main again, theres some conflicts again